src/main/groovy/com/instabot/utils/filehandler/FileHandler.groovy

```diff
@@ -9,6 +9,7 @@ import java.time.LocalDateTime
 class FileHandler {
     private static final Logger LOG = LogManager.getLogger(FileHandler.class)
 
+    //TODO Just logging the error and the page source to the normal log file would be ok too
```

Comment:
Is there any specific concern about using it this way?
The idea behind creating a separate file is that it can be open (w/o the effort of extracting a huge chunk of data from the log file manually) to visualize how the web-page looked like at the moment of the issue and e.g., use (crhome/firefox) dev tools to inspect the problem